src/widgets/medications/medication-order.component.tsx

```diff
@@ -375,13 +376,14 @@ export default function MedicationOrder(props: MedicationOrderProps) {
             <div className={styles.medicationOrderInput}>
               <label htmlFor="startDate">Start date</label>
               <input
-                type="text"
+                type="Date"
```

Comment:
HTML is pretty resilient here, but we should still use the lowercase `date` here.